src/Acorn/Sage/Sage.php

```diff
@@ -13,53 +13,153 @@
+     */
     protected $app;
 
-    /** @var \Roots\Acorn\Sage\ViewFinder */
-    protected $sage_finder;
+    /**
+     * The ViewFinder instance.
+     *
+     * @var \Roots\Acorn\Sage\ViewFinder
+     */
+    protected $sageFinder;
 
-    /** @var \Roots\Acorn\View\FileViewFinder */
-    protected $file_finder;
+    /**
+     * The FileViewFinder instance.
+     *
+     * @var \Roots\Acorn\View\FileViewFinder
+     */
+    protected $fileFinder;
 
-    /** @var \Illuminate\Contracts\View\Factory */
+    /**
+     * The View Factory instance.
+     *
+     * @var \Illuminate\Contracts\View\Factory
+     */
     protected $view;
 
+    /**
+     * Creates a new Sage instance.
+     *
+     * @param Filesystem        $files
+     * @param ViewFinder        $sageFinder
+     * @param FileViewFinder    $fileFinder
+     * @param ViewFactory       $view
+     * @param ContainerContract $app
+     */
     public function __construct(
         Filesystem $files,
-        ViewFinder $sage_finder,
-        FileViewFinder $file_finder,
+        ViewFinder $sageFinder,
+        FileViewFinder $fileFinder,
         ViewFactory $view,
         ContainerContract $app
     ) {
         $this->app = $app;
         $this->files = $files;
-        $this->file_finder = $file_finder;
-        $this->sage_finder = $sage_finder;
-        $this->view = $view;
+        $this->fileFinder = $fileFinder;
+        $this->sageFinder = $sageFinder;
+        $this->views = $view;
     }
 
+    /**
+     * Initialize and attach Sage features.
+     *
+     * @return void
+     */
     public function attach()
     {
         $this->attachPostVariable();
         $this->attachCommentsTemplateResolver();
+        $this->attachSearchFormFilter();
+        $this->attachBodyClassFilter();
         $this->attachTemplateHierarchyFilters();
         $this->attachTemplateIncludeFilter();
-        $this->attachThemeTemplateFilter();
+        $this->attachThemeTemplatesFilter();
     }
 
+    /**
+     * Attach global `$post` variable to Blade views.
+     *
+     * @param  integer $priority
+     * @return void
+     */
     public function attachPostVariable($priority = 10)
     {
         add_action('the_post', function ($post) {
             $this->view->share('post', $post);
         }, $priority);
     }
 
+    /**
+     * Search for a compiled Blade partial when resolving the comments template.
+     *
+     * @param  integer $priority
+     * @return void
+     */
     public function attachCommentsTemplateResolver($priority = 100)
     {
-        add_filter('comments_template', [$this, 'commentsTemplateResolver'], $priority);
+        add_filter('comments_template', function ($name) {
+            if (Str::startsWith($file, [STYLESHEETPATH, TEMPLATEPATH])) {
+                $file = ltrim(str_replace([STYLESHEETPATH, TEMPLATEPATH], '', $file), '\\/');
+            }
+
+            if ($file == '/comments.php') {
+                $file = '/partials/comments.blade.php';
```

Comment:
This should be `'partials/comments.blade.php'` to be consistent with `ltrim()` above and because it's a relative path.